content/Foundation/Trademark/home.pml

```diff
@@ -12,8 +12,8 @@
   in your work, following the usage rules below.
 </p>
 <p>
-  The Linux Foundation Projects (LF Projects, LLC) has applied to register the
-  seL4 trademark in the United States and other countries.
+   The Linux Foundation has registered seL4<sup>&reg;</sup> as a
+  trademark in the United States. The Linux Foundation Projects (LF Projects, LLC) has also applied to register the seL4 trademark in other countries.
```

Comment:
Ah, sorry, misunderstanding. I meant we can remove: "The Linux Foundation Projects (LF Projects, LLC) has also applied to .." .

The first sentence should instead be: "The Linux Foundation has registered seL4<sup>&reg;</sup> as a   trademark in the United States and other countries".

I'm easy on whether to include the "Please use TM sign" sentence or not. It is in the guidelines that are being linked in the footer, but if people care a lot about it, I'm fine to repeat it in the footer itself.

Reply:
Ok. First sentence updated.
I'll leave out the sentence about using the TM sign. It was commented out in the pml, so I'll leave it as a comment unless someone objects.